amo2kinto/exporter.py

```diff
@@ -83,16 +83,17 @@ def write_addons_items(xml_tree, records, app_id, api_ver=3):
         if is_related_to(item, app_id):
             if item['guid'] in groupby:
                 emItem = groupby[item['guid']]
-                # Remove the first caracter which is the letter i to
-                # compare the numeric value i45 < i356.
-                current_blockID = item['blockID'][1:]
-                previous_blockID = emItem.attrib['blockID'][1:]
-                # Group by and keep the biggest blockID in the XML file.
-                if current_blockID > previous_blockID:
-                    emItem.attrib['blockID'] = item['blockID']
+                if 'blockID' in item:
```

Comment:
A small comment explaining that in the new system `blockID` is not mandatory, but `id` is.